sale_resource_booking/views/resource_booking_views.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2021 Tecnativa - Jairo Llopis
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<data>
+    <record id="resource_booking_view_form" model="ir.ui.view">
+        <field name="name">Display sale order info in booking form</field>
+        <field name="model">resource.booking</field>
+        <field name="inherit_id" ref="resource_booking.resource_booking_view_form" />
+        <field name="arch" type="xml">
+            <header position="inside">
+                <button
+                    name="action_generate"
+                    string="Quote"
+                    type="object"
+                    help="Generate quotation."
+                />
+            </header>
+            <group name="booking" position="inside">
+                <field
+                    name="product_id"
+                    domain="[
+                        ('resource_booking_type_id', '=', type_id),
+                        '|',
+                        ('resource_booking_type_combination_rel_id', 'in', [combination_id]),
+                        ('resource_booking_type_combination_rel_id', '=', None),
+                    ]"
+                />
+            </group>
+            <group name='main' position='inside'>
+                <group
+                    name="sale"
+                    string="Sales"
+                    groups_id="[(4, ref('sales_team.group_sale_salesman'), 0)]"
+                >
+                    <field name="sale_order_line_id" invisible="1" />
+                    <field
+                        name="sale_order_line_ids"
+                        invisible="1"
+                        widget="many2many_tags"
+                    />
+                    <field name="sale_order_id" />
+                    <field name="sale_order_state" />
+                </group>
+            </group>
+        </field>
+    </record>
+    <record id="resource_booking_view_tree" model="ir.ui.view">
+        <field name="name">resource.booking.tree.inherit</field>
+        <field name="model">resource.booking</field>
+        <field name="inherit_id" ref="resource_booking.resource_booking_view_tree" />
+        <field name="arch" type="xml">
+            <field name="partner_id" position="after">
```

Comment:
i think that the correct field here, is partner_ids

Reply:
Done